CloudOnMobile/Modules/Files/Models/FileType.swift

```diff
@@ -14,6 +14,27 @@ enum FileType {
     case other
 }
 
+extension FileType {
+    init(filename: String) {
+        let pathExtension = URL(fileURLWithPath: filename).pathExtension.lowercased()
+        switch pathExtension {
+        case "gif",
+             "jpeg",
+             "jpg",
+             "png":
+            self = .image
+        case "mp3",
+             "wav":
+            self = .music
+        case "mov",
+             "mp4":
+            self = .video
+        default:
+            self = .other
+        }
```

Comment:
What do you think about creating a property:
```
var pathExtension: [String] {
     switch self {
        case .image: return ["gif", ..., ...]
       ....
       ...
     }
}
```

In this way we will be required to handle all cases by compiler.
initializer will look like:
```
FileType.allCases.first(where: { $0. pathExtension == value }) ?? .other
```